open-company/open-company-web
# Fix login redirect

BUG: if you try to login with the wrong password you get redirected to the homepage. This is because the failed login return a 401 and we force the user to logout on every 401 response from our services since that means the token is no longer valid.

Solution: i added a check so the force logout and redirect happens only if the user is actually logged in, this should keep the old behaviour but avoid redirect when the user tries to login

To test:
- signup with email
- finish onboard
- logout
- try logging in with a wrong password
- [x] you still on the login form?
- [x] do you see a failed login error?
- now try to login with the real password
- now with dev tools go and change the `jwt` cookie (it's called `localhost-jwt` or `staging-jwt` or `jwt` depending on the env)
- replace it with a random string
- refresh the page
- [x] did it NOT load?
